### PR TITLE
new stuff idfk what xoxo

### PR DESCRIPTION
**Pull request by JamesHawkinss**
# Pull request

 - [x] I have read and agreed to the [CODE OF CONDUCT](https://github.com/CascadeBot/CascadeBot/blob/master/CODE_OF_CONDUCT.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [trello](https://trello.com/b/gUQehy4l/bot).
 - [x] I have tested all of my changes.
 - [x] I have followed the style of the rest of the project.
 
## Added/Changed
What did you add and/or change for this.

## Feature
If this is a new feature, why should we add it to the bot.

### CODE REVIEW

File: src/main/java/com/cascadebot/cascadebot/commands/informational/UserInfoCommand.java [hunk 1]
```diff
@@ -39,24 +40,38 @@ public void onCommand(Member sender, CommandContext context) {
...
+        if (memberForInfo.getOnlineStatus() == OnlineStatus.ONLINE) {
+            status = context.globalEmote("online");
+        } else if (memberForInfo.getOnlineStatus() == OnlineStatus.OFFLINE) {
+            status = context.globalEmote("offline");
+        } else if (memberForInfo.getOnlineStatus() == OnlineStatus.DO_NOT_DISTURB) {
+            status = context.globalEmote("dnd");
+        } else if (memberForInfo.getOnlineStatus() == OnlineStatus.IDLE) {
+            status = context.globalEmote("idle");
+        } else {
+            status = context.globalEmote("offline");
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/informational/UserInfoCommand.java]: Remove this else statement, you've already set it to ""

Comment (JamesHawkinss) [reply src/main/java/com/cascadebot/cascadebot/commands/informational/UserInfoCommand.java]: kk xoxo

File: src/main/java/com/cascadebot/cascadebot/commands/informational/ServerInfoCommand.java [hunk 2]
```diff
@@ -36,12 +36,12 @@ public void onCommand(Member sender, CommandContext context) {
         builder.setThumbnail(guildForInfo.getIconUrl());
         builder.addField("Creation Date", FormatUtils.formatDateTime(guildForInfo.getCreationTime()), true);
         builder.addField("Guild Name", guildForInfo.getName(), true);
-        builder.addField("Owner", guildForInfo.getOwner().getUser().getAsTag(), true);
+        builder.addField("Owner" + context.globalEmote("server_owner"), guildForInfo.getOwner().getUser().getAsTag(), true);
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/informational/ServerInfoCommand.java]: Add a space

File: src/main/java/com/cascadebot/cascadebot/commands/informational/UserInfoCommand.java [hunk 3]
```diff
@@ -39,24 +40,36 @@ public void onCommand(Member sender, CommandContext context) {
...
+
         List<Page> pageList = new ArrayList<>();
         EmbedBuilder builder = MessagingObjects.getInfoEmbedBuilder();
         builder.setTitle(user.getAsTag());
         builder.setThumbnail(user.getAvatarUrl());
         builder.addField("User Created", user.getCreationTime().format(DateTimeFormatter.RFC_1123_DATE_TIME), true);
         builder.addField("Join Date", memberForInfo.getJoinDate().format(DateTimeFormatter.RFC_1123_DATE_TIME), true);
         builder.addField("User ID", user.getId(), true);
-        builder.addField("Status", StringUtils.capitalize(memberForInfo.getOnlineStatus().toString().replace("_", " ").toLowerCase()), true);
+        builder.addField("Status", StringUtils.capitalize(memberForInfo.getOnlineStatus().toString().replace("_", " ").toLowerCase()) + "  " + status, true);
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/informational/UserInfoCommand.java]: Why two spaces?

**Pull request merged**